generators/client-2/templates/src/main/webapp/app/shared/_shared.ng2module.ts

```diff
@@ -24,12 +34,19 @@ import {
 @NgModule({
     imports: [
         <%=angular2AppName%>SharedLibsModule,
-        <%=angular2AppName%>SharedCommonModule,
+        <%=angular2AppName%>SharedCommonModule<%_ if (enableTranslation){ _%>,
+        TranslateModule.forRoot({
```

Comment:
Ok I just read the docs, so it can be in a shared module. In that case better add it to the `<%=angular2AppName%>SharedLibsModule` and add the custom translation related directives/pipes in the `SharedCommonModule` as this module imports both those